system/doc/reference_manual/expressions.xml

```diff
@@ -470,13 +472,15 @@ after
     ExprT ->
         BodyT
 end</pre>
-    <p><c>ExprT</c> is to evaluate to an integer. The highest allowed
-      value is 16#FFFFFFFF, that is, the value must fit in 32 bits.
-      <c>receive..after</c> works exactly as <c>receive</c>, except
+    <p><c>receive..after</c> works exactly as <c>receive</c>, except
       that if no matching message has arrived within <c>ExprT</c>
       milliseconds, then <c>BodyT</c> is evaluated instead. The
       return value of <c>BodyT</c> then becomes the return value
-      of the <c>receive..after</c> expression.</p>
+      of the <c>receive..after</c> expression. <c>ExprT</c> is to
+      evaluate to an integer. The highest allowed value is
```

Comment:
```suggestion
      evaluate to an integer, or the atom <c>infinity</c>. If it evaluates to an integer, the highest allowed value is
```

Reply:
Hmm, that's weird. I think it is quite confusing to first claim that it should evaluate to an integer and then half a page below add to it. It should probably also describe the allowed range of integers, `i.e.` 0..4294967295`, not just mention the largest value allowed.